ddp-workspace/projects/ddp-dsm-ui/src/app/services/dsm.service.ts

```diff
@@ -288,6 +288,27 @@ export class DSMService {
     );
   }
 
+  public createCohortTag(body: string, realm: string): Observable<any> {
+    const url = this.baseUrl + DSMService.UI + 'createCohortTag';
+    const map: { name: string; value: any }[] = [];
+    map.push({name: 'userId', value: this.role.userID()});
+    map.push({name: 'realm', value: realm});
```

Comment:
**_#minor_**

Just wondering why not like this? 
```
const map = [
    { name: 'userId', value: this.role.userID() }, 
    { name: 'realm', value: realm }
];
```

Same for `deleteCohortTag` method below.